tools/mir-smoke-test-runner.sh

```diff
@@ -5,9 +5,18 @@ date --utc --iso-8601=seconds | xargs echo "[timestamp] Start time :"
 
 mir_rc=0
 timeout=3
-WAYLAND_DESKTOP=mir-smoke-test
-MIR_SERVER_WAYLAND_SOCKET=${WAYLAND_DESKTOP}
 
+if [ -v MIR_SOCKET ]
+then
+  if [ ! -e "${MIR_SOCKET}" ]
```

Comment:
Should we not do the same for WAYLAND_DISPLAY?

Reply:
No. We ignore the desktop's WAYLAND_DISPLAY and set up another for the test server and client.